### DISCUSSION TITLE
📸  Built with React Native Firebase showcase

### DISCUSSION THREAD

**Discussion by Ehesp** [invertase/react-native-firebase#4106] [Show and tell]
We're looking to showcase Android and/or iOS applications on our website 🥳 If you're using React Native Firebase please let us know!

Please add a comment below 🔥

---

- **Name / Company**:
- **Application / Product Name:**
- **Play Store URL**:
- **App Store URL**:
- **Modules** (e.g: firestore, analytics):
- **Additional details**:

**Comment by AndrewJack** [comment]
Name / Company: Digital Tribe

**Application / Product Name:**  DriveTribe
**Play Store URL:** https://play.google.com/store/apps/details?id=com.drivetribe&hl=en
**App Store URL:** https://apps.apple.com/gb/app/drivetribe/id1164159977
**Modules:** analytics, crashlytics, in-app-messaging, messaging, perf, remote-config, dynamic-links
**Additional details:** An automotive online community platform founded by The Grand Tour presenters Jeremy Clarkson, James May and Richard Hammond.

**Application / Product Name:**  FoodTribe
**Play Store URL:** https://play.google.com/store/apps/details?id=com.foodtribe.app&hl=en_US
**App Store URL:** https://apps.apple.com/us/app/foodtribe-app-for-foodies/id1478284014
**Modules:** analytics, crashlytics, in-app-messaging, messaging, perf, remote-config, dynamic-links
**Additional details:** A food based online community platform founded by Jeremy Clarkson, James May and Richard Hammond.

**Comment by ahanusek** [comment]
Name / Company: fireup.pro

Application / Product Name: treneiro.me 
Play Store URL: https://play.google.com/store/apps/details?id=com.fireup.pro.treneiro
App Store URL: https://apps.apple.com/gb/app/treneiro-me/id1397018823
Modules: analytics, auth, crashlytics, dynamic-links, firestore, functions, messaging, perf, remote-config, storage
Additional details: App for personal trainers and their clients (workouts tracking, nutrition plans, body & weight measurements, and many more).

**Comment by diegolmello** [comment]
**Name / Company:** Rocket.Chat
**Application / Product Name:** Rocket.Chat
**Play Store URL:** https://play.google.com/store/apps/details?id=chat.rocket.android
**App Store URL:** https://apps.apple.com/us/app/rocket-chat/id1148741252
**Modules:** analytics, crashlytics
**Additional details:** The ultimate Free Open Source Solution for team communications.
Server repo: https://github.com/RocketChat/Rocket.Chat
App repo: https://github.com/RocketChat/Rocket.Chat.ReactNative

**Comment by shirakaba** [comment]
**Name / Company:** Jamie Birch
**Application / Product Name:** LinguaBrowse
**~~Play Store URL:~~**
**App Store URL:** https://apps.apple.com/gb/app/linguabrowse/id1281350165
**Modules:** Auth, Dynamic Links, Firestore, Analytics
**Additional details:** 🌐 The ultimate web browser for studying foreign languages. 📖 Easily read any foreign-language website with the help of a one-tap popup dictionary; 🔤 add transcriptions to languages such as Japanese and Chinese; 🗣 hear pronunciations upon tapping words; 📝 and automatically build a vocabulary list of all the words that have challenged you. 📣 _Twitter: [@LinguaBrowse](https://twitter.com/LinguaBrowse)_

**Comment by rahul2992** [comment]
**Name / Company:** HTT Holidaying Travel Technologies Pvt. Ltd.
**Application / Product Name:** Holidaying.travel 
**Play Store URL:** https://play.google.com/store/apps/details?id=com.holidaying.holidaying&hl=en
**App Store URL:** https://apps.apple.com/in/app/holidaying/id1497348229
**Modules (e.g: firestore, analytics):** firestore, auth, app, analytics, storage, crashlytics, dynamic-links
**Additional details:** We are a platform for people to discover, plan and book trips by travel influencers and locals. Anyone from a blogger, influencer or an avid traveller can create their travel blog/store on the app, and earn for any bookings. For travellers, it gives a one step solution from reading a blog to actually booking and planning the trip. 

Website: https://holidaying.travel/influencer/

**Comment by mikehardy** [comment]
- **Name / Company**: Kullki Fintech KSS S.A.
- **Application / Product Name:** [Komp.ai](https://komp.ai)
- **Play Store URL**: https://play.google.com/store/apps/details?id=com.kullki.kscore
- **App Store URL**: https://apps.apple.com/us/app/komp-ai/id1498259800?ls=1
- **Modules** (e.g: firestore, analytics): firestore, messaging, analytics, dynamic links, auth, crashlytics
- **Additional details**: Komp.ai is an award-winning, secure, privacy-focused technological solution focused on non-conventional data mining based on your mobile device information. This data can be used when making financial decisions and even predict behavior based on our KompAI score.

**Comment by andersonaddo** [comment]
**Name**: Anderson Addo
**Application**: Biteup
**App Store URL**: pending
**Play Store URL**:  pending
**Modules**: Firestore, Messaging, Performance, Crashlytics, Analytics, Auth, Database, Storage
**Additional Details**: Biteup is a social app that is designed to being people physically together over food. Release delayed by COVID (will update comment when situation changes). Website: https://biteup.app

**Comment by DanGDroid** [comment]
**Name / Company:** Kimaia
**Application / Product Name:** [FARMER](https://www.farmerdirecto.com/)
**Play Store URL:** https://play.google.com/store/apps/details?id=com.agricula.farmer
**App Store URL:** https://apps.apple.com/il/app/farmer-direct/id1327728378
**Modules:** Analytics, Messaging, Notifications, Dynamic-Links;
**Future Modules**: A/B testing, Remote-Config
**Additional details:** FARMER is great news for farmers, and even bigger news for consumers. Because FARMER is a place where the farmer and the end consumer meet. That’s what we call “Direct Farming”.

**Comment by r0b0t3d** [comment]
**Name / Company:** STT LABS CO., LTD
**Application / Product Name:** [Stuneo](https://stuneo.com/)
**Play Store URL:** https://play.google.com/store/apps/details?id=com.stuneo
**App Store URL:** https://apps.apple.com/vn/app/stuneo-3d-video-for-run-ride/id1485979882
**Modules:** Firestore, Auth, Crashlytics, Analytics, Messaging, Dynamic-Links, Storage, Functions
**Additional details:** Stuneo is a FREE app helps to create 3D landscape video of your Strava outdoor activities such as: Running, Riding, Hiking & More. You connect with Strava account to import activities and then the app will generate 3D videos of bird-eye view for you automatically. You can also track your run, ride or hike in-app. The video then can be shared with friends & family.

**Comment by tuan072090** [comment]
Name / Company: Meete
Application / Product Name: Meete
Play Store URL: https://play.google.com/store/apps/details?id=com.cixa.meete
App Store URL: https://apps.apple.com/vn/app/meete/id1066923602
Modules: Auth, Crashlytics, Analytics, Messaging, Dynamic-Links, Remote config, A/B testing
Additional details: Meete is an e-voucher app that helps people can experience quality restaurants with the best promotion.

**Comment by laurentS** [comment]
Name / Company: MapSwipe
Application / Product Name: MapSwipe
Play Store URL: https://play.google.com/store/apps/details?id=org.missingmaps.mapswipe
App Store URL: https://apps.apple.com/us/app/mapswipe/id1133855392
Modules (e.g: firestore, analytics): Realtime Database, Auth, Analytics, Cloud Messaging
Additional details: MapSwipe lets users label satellite imagery to help humanitarians find and help vulnerable people.

**Comment by ACHP** [comment]
**Name / Company:** GensDeConfiance
**Application / Product Name:** GensDeConfiance
**Play Store URL:** https://play.google.com/store/apps/details?id=fr.gensdeconfiance.app
**App Store URL:** https://apps.apple.com/us/app/gdc-petites-annonces/id1103739976
**Modules :** Crashlytics, Analytics, Messaging
**Additional details:** Your ads on a trustworthy network. Tired of online scams? Time to try GensDeConfiance! With more than 500 000 members, GensDeConfiance offers a trustworthy and secured way to sell, buy, and rent with other individuals. Give it a try, it’s free!

**Comment by davidepalazzo** [comment]
**Name / Company:** The 'drobe
**Application / Product Name:** [The 'drobe - Outfit Planner](https://thedrobe.com)
**Play Store URL:** https://play.google.com/store/apps/details?id=com.thedrobe
**App Store URL:** https://apps.apple.com/app/the-drobe/id1448655936?ls=1
**Modules:** Analytics, Authentication, Functions, Storage, Crashlytics, Realtime Database, Dynamic Links, Cloud Messaging, Hosting
**Additional details:** The 'drobe is a cross-platform app & web app that helps people interested in sustainable and/or minimalist fashion manage their closet.

**Comment by photorouletteadmin** [comment]
**Application / Product Name**: Photo Roulette
**App Store URL**: https://apps.apple.com/app/id1050443738
**Play Store URL**: https://play.google.com/store/apps/details?id=com.photoroulette
**Modules**: Authentication, Cloud Functions, Cloud Storage, Crashlytics, Realtime Database, Messaging, Performance Monitoring, Remote Config

**Comment by MujtabaFR** [comment]
Application / Product Name: **Capsula**
Play Store URL: https://play.google.com/store/apps/details?id=com.swiftdeer2.capsula
App Store URL: https://apps.apple.com/us/app/id1450718866
Modules: analytics, auth, crashlytics, dynamic-links, firestore, cloudfunctions, messaging, perf, storage
Additional details: Specialized in suppling pharmacies in Iraq from the warehouses

**Comment by mitchellmcm27** [comment]
Name / Company: Cool Font LLC
Application / Product Name: Mixel
Play Store URL: https://play.google.com/store/apps/details?id=com.cfdrink
App Store URL: https://itunes.apple.com/app/id1280464759
Modules: Firestore, analytics, auth, dynamic links, crashlytics, cloud functions, storage
Additional details: Cocktail recipe app with >1k daily active users, https://www.mixelcocktails.com

**Comment by psdewar** [comment]
**Application / Product Name**
Lyrist

**Website**
https://lyrist.app

**Play Store URL**
https://lyrist.app/android

**App Store URL**
https://lyrist.app/ios

**Modules**
Analytics, auth, cloud functions, dynamic links, firestore, with plans to leverage realtime database for write-heavy operations

**Additional details**
- Write text while listening to audio (YouTube) on the same screen
- Perfect for songwriters, students, translators, and anyone who takes notes
- \> 150 daily active users

**Comment by rahul2992** [comment]
**Name / Company:**: SocialBoat
**Application / Product Name:** SocialBoat
**Play Store URL:** https://play.google.com/store/apps/details?id=com.socialboat.socialboat&gl=US
**App Store URL:** https://apps.apple.com/sg/app/socialboat/id1635586100
**Website:** https://socialboat.live/women
**Modules (e.g: firestore, analytics):** Firestore, Auth, Dynamic Links, Messaging
**Additional details:** AI Powered fitness game for women.

**Comment by ghost** [comment]
Name / Company: [Aurum](https://www.aurum.com.br/) 
Application / Product Name: Astrea
Play Store URL: https://play.google.com/store/apps/details?id=br.net.astrea.app&hl=pt_BR&gl=US&pli=1
App Store URL: https://apps.apple.com/br/app/astrea-software-jur%C3%ADdico/id1498206479
Modules (e.g: firestore, analytics): analytics, crashlytics, database, messaging and remote-config.
Additional details: APP of the largest legal system in Latin America
